build.sbt

```diff
@@ -1,14 +1,15 @@
 import org.openqa.selenium.chrome.ChromeOptions
-import org.openqa.selenium.remote.DesiredCapabilities
-import org.scalajs.jsenv.selenium.SeleniumJSEnv
 
 name := "udash-jquery"
 
 inThisBuild(Seq(
-  version := "2.0.1",
+  version := "3.0.0-SNAPSHOT",
```

Comment:
You'll change it before releasing, right? ;)

Reply:
Right :)